spec/jobs/update_account_invoices_spec.rb

```diff
@@ -157,7 +157,7 @@ def travel_to(time)
         context "where the order is not complete" do
           before do
             allow(invoice_order).to receive(:complete?) { false }
-            june_billable_period1.enterprise.update_attributes(contact: "Firstname Lastname Something Else", phone: '12345')
```

Comment:
Keep

Reply:
Reinstated in 0d57da3 and renamed to `contact_name`.